src/test/java/bio/overture/ego/grpc/service/UserServiceGrpcTest.java

```diff
@@ -79,6 +80,8 @@ public void setUpInProcessGrpc() throws IOException {
   @Before
```

Comment:
same as other test, unless there is a specific reason for using `@Before` twice, it would be best to create
```java
@Before
public void init(){
  before();
  setUpTestData();
}
```